azerothcore/azerothcore-wotlk
# fix(DB/creature_loot_template) Increase droprate for item Healthy Dragon Scale

## Changes Proposed:
-  Increase droprate of [Healthy Dragon Scale](https://classic.wowhead.com/item=13920/healthy-dragon-scale) from 6% to 20%

## Issues Addressed:
This is affecting quest [Healthy Dragon Scale](https://classic.wowhead.com/quest=5582/healthy-dragon-scale) which is only available once you finish the [Plagued Hatchlings Quest](https://classic.wowhead.com/quest=5529/plagued-hatchlings).

- The AC database has a droprate setting of 6%
- The TC database has a droprate of 10%
- These guys went for 100% : https://wotlk-twinhead.twinstar.cz/?npc=10678
- Here the rate is set to ~21% : https://classic.wowhead.com/item=13920/healthy-dragon-scale 
- The comments indicate that the drop rate was low in vanilla - 3-5 scales for a whole run.
- A comment from TBC mentions they had 12 scales in a run.

So this is all RNG and so on and so forth, but 6% is probably a bit too low while 100% is also too much.

**My compromise is 20% unless anyone knows a more accurate rate!**



## Tests Performed:
- Applied SQL, killed dragons and counted scales



## How to Test the Changes:

- `.quest add 5529`
- `.quest complete 5529`
- `.quest reward 5529`
- `.go c id 10678`
- kill and loot hatchlings - since this is RNG, you might not see a change every time


## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
